diesel/src/pg/expression/functions.rs

```diff
@@ -106,3 +106,51 @@ define_sql_function! {
     #[cfg(feature = "postgres_backend")]
     fn lower<T: RangeHelper>(range: T) -> Nullable<<T as RangeHelper>::Inner>;
 }
+
+define_sql_function! {
+    /// Returns range of integer.
+    /// if the range is empty or has no lower bound, it returns NULL.
```

Comment:
This doesn't sound correct?

Reply:
I copied and past and forgot to change this line, sorry